alica_engine/include/engine/PlanInterface.h

```diff
@@ -73,8 +73,6 @@ class ThreadSafePlanInterface
     // Non locked thread safe reads:
     // Reads of const members are thread safe (rp's lifetime is deemed long enough):
 
-    AlicaEngine* getAlicaEngine() const { return _rp->getAlicaEngine(); }
```

Comment:
should be able to just delete this entire file (& the .cpp) as there shouldn't be any references to this class anywhere in the engine or tests -> can you confirm & remove it if so?